modules/ztp-preparing-the-hub-cluster-for-ztp.adoc

```diff
@@ -19,7 +19,15 @@ $ oc get secret openshift-gitops-cluster -n openshift-gitops -o jsonpath='{.data
 ----
 
 . Prepare the ArgoCD pipeline configuration:
-.. Clone the Git repository.
+.. Extract the ArgoCD deployment CRs from the ZTP site generator container using the latest container image version:
++
+[source,terminal]
+----
+$ mkdir ztp
+$ podman run --rm -v `pwd`/ztp:/mnt/ztp:Z registry.redhat.io/openshift4/ztp-site-generate-rhel8:v4.9.0-1 /bin/bash -c "cp -ar /usr/src/hook/ztp/* /mnt/ztp/"
+----
++
+The remaining steps in this section relate to the `ztp/gitops-subscriptions/argocd/` directory.
```

Comment:
> The remaining steps in this section relate to the `ztp/gitops-subscriptions/argocd/` directory.

I am getting tripped up due to the context as part of the `3a` task.

Does it make sense to change this into a substep (3b) and have the customer navigate to the `ztp/gitop-subscriptions/argocd/` directory?

Or simply add the full file path to the files below?

Reply:
Thanks Michael. I think your second suggestion is the clearer option.
i will change the instances as follows: 
<1> `path` is the ztp/gitops-subscriptions/argocd/ file path that contains the `policygentemplates` CRs for the clusters.